pkg/operator/targetconfigcontroller/targetconfigcontroller_test.go

```diff
@@ -155,11 +181,15 @@ func TestTargetConfigController(t *testing.T) {
 				operatorClient:        fakeOperatorClient,
 				kubeClient:            fakeKubeClient,
 				envVarGetter:          envVar,
-				backupVarGetter:       backuphelpers.NewDisabledBackupConfig(),
+				backupVarGetter:       backupVar,
 				enqueueFn:             func() {},
 				quorumChecker:         quorumChecker,
 			}
 
+			if scenario.etcdBackupSpec != nil {
```

Comment:
@tjungblu  I added this scenario 

Since we dont error upon `BackupVar` I dont know what to assert upon 🤔

Reply:
the output of the static pod, not sure why this was never added here